mindjunkies/forums/views.py

```
@@ -99,3 +105,24 @@
         return HttpResponseRedirect(
             f"{reverse('forum_home', kwargs={'course_slug': course_slug})}#replies-container-{topic_id}"
         )
+    def handle_reaction_submission(self, request, course_slug):
+        topic_id = request.POST.get("topic_id")
+        topic = get_object_or_404(ForumTopic, id=topic_id)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)